knexfile.js

```diff
@@ -18,6 +18,9 @@ module.exports = {
     },
     migrations: {
       directory: __dirname + '/migrations'
+    },
```

Comment:
Needed to manually add this as well as migrations: doesn't get set up with knex. Good to know for future reference.